src/components/CategoryList.js

```diff
@@ -27,24 +27,27 @@ export class CategoryList extends React.PureComponent {
   };
 
   render() {
-    const { data, navigation, noSubtitle, refreshControl } = this.props;
+    const { data, navigation, noSubtitle, refreshControl, noSectionHeader } = this.props;
+
+    // Sorting data alphabetically
+    data.sort((a, b) => a.title.localeCompare(b.title));
 
     const sectionedData = [
       {
         title: texts.categoryTitles.pointsOfInterest,
-        data: _filter(data, (category) => category.pointsOfInterestCount > 0)
+        data: _filter(data, (category) => category.pointsOfInterestCount > 0 && !category.parent)
       },
       {
         title: texts.categoryTitles.tours,
-        data: _filter(data, (category) => category.toursCount > 0)
+        data: _filter(data, (category) => category.toursCount > 0 && !category.parent)
       }
     ];
 
     return (
       <SectionList
         keyExtractor={this.keyExtractor}
         sections={sectionedData}
-        initialNumToRender={data.length}
+        initialNumToRender={data.length > 1 ? data.length : 2}
```

Comment:
why is this condition necessary here? could you add a comment to explain right there at the code?

Reply:
If data.length is 1, it is not displayed on the screen. Section list needs more than 1 data.